docs/content/start/telemetry.md

```diff
@@ -0,0 +1,255 @@
+
+
+```ts
+public async createContainer(
+  serviceContainerConfig: TinyliciousContainerConfig,
+  containerSchema: ContainerSchema,
+): Promise<[container: FluidContainer, containerServices: TinyliciousContainerServices]>
+```
+`createContainer()` interface definition takes a `TinyliciousContainerConfig` type as its `serviceContainerConfig` argument. (The `FrsClient` will take `FrsContainerConfig` for its respective methods)
+
+```ts
+static async getContainer(
+  serviceConfig: TinyliciousContainerConfig,
+  objectConfig: ContainerSchema,
+): Promise<[container: FluidContainer, containerServices: TinyliciousContainerServices]>
+```
+`getContainer()` interface definition takes a `TinyliciousContainerConfig` type as its `serviceContainerConfig` argument. (The `FrsClient` will take `FrsContainerConfig` for its respective methods)
+
+
+```ts
+const loader = new Loader({
+  urlResolver: this.urlResolver,
+  documentServiceFactory: this.documentServiceFactory,
+  codeLoader,
+  logger: tinyliciousContainerConfig.logger,
+});
+```
+The `Loader` object is called by both `createContainer()` and `getContainer()`, and requires a `ILoaderProps` interface as its constructor argument. `ILoaderProps` interface has an optional logger parameter that will take the `ITelemetryBaseLogger` defined by the user.
+
+```ts
+constructor(loaderProps: ILoaderProps) {
+  const scope = { ...loaderProps.scope };
+  if (loaderProps.options?.provideScopeLoader !== false) {
+      scope.ILoader = this;
+  }
+
+  this.services = {
+      urlResolver: createCachedResolver(MultiUrlResolver.create(loaderProps.urlResolver)),
+      documentServiceFactory: MultiDocumentServiceFactory.create(loaderProps.documentServiceFactory),
+      codeLoader: loaderProps.codeLoader,
+      options: loaderProps.options ?? {},
+      scope,
+      subLogger: DebugLogger.mixinDebugLogger("fluid:telemetry", loaderProps.logger, { all:{loaderId: uuid()} }),
+      proxyLoaderFactories: loaderProps.proxyLoaderFactories ?? new Map<string, IProxyLoaderFactory>(),
+      detachedBlobStorage: loaderProps.detachedBlobStorage,
+  };
+  this.logger = ChildLogger.create(this.services.subLogger, "Loader");
+}
+```
+`ILoaderProps.logger` is used by `Loader` to pipe to container's telemetry system.
+
+
+
+### Properties and methods
+
+The interface contains a `supportTags` property and a `send()` method as shown:
+
+```ts
+export interface ITelemetryBaseLogger {
+  supportsTags?: true;
+  send(event: ITelemetryBaseEvent): void;
+}
+```
+
+- supportsTags
+  - Tags are strings used to classify different events. In a simple logger, all events are untagged and handled the same by your logger's implementation. However, in some scenarios, where some data should be handled separately (for example, private customer data), those events could be tagged with a unique string so they could be treated differently downstream.
```

Comment:
Just cut/paste this paragraph down to the section about `ITelemetryBaseEvent` to give context about the tagged property type.